force-app/flowcomponents/lwc/nksFlowNavigation/templateWithFooter.html

```diff
@@ -2,6 +2,7 @@
     <footer class="slds-card__footer" style="width: 100%">
         <lightning-layout horizontal-align={alignment}>
             <lightning-button
+                class="buttonClass"
```

Comment:
Her må du bruke curly brackets, enkel typo 
```suggestion
                class={buttonClass}
```